### PR TITLE
Update according to nf-core pipeline guidelines

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

This PR includes changes that align with the guidelines at https://nf-co.re/docs/contributing/pipeline_release_review_guidelines

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules/annotate_consequence_pli.config
Comment (jemten) [inline conf/modules/annotate_consequence_pli.config:1]: Later we should move these two process to the annotate snv/sv subworkflows

File: docs/usage.md [hunk 1]
```diff
@@ -246,8 +246,8 @@ nextflow run nf-core/raredisease \
     -profile test,<YOURPROFILE> \
     --input samplesheet.csv \
     --fasta reference.fasta \
-    --annotate_snv_switch false \
-    --annotate_sv_switch false \
+    --skip_snv_annotation true \
+    --skip_sv_annotation true \
```
Comment (jemten) [inline docs/usage.md:250]: Same comment as above

**Review by jemten — COMMENTED** [review]
Nice work with the `skip` switches and tidying up the use of hardcoded prefixes in local modules. Some very small comments

File: conf/modules/annotate_snvs.config [hunk 2]
```diff
@@ -44,7 +48,7 @@ process {
 
     withName: '.*ANNOTATE_SNVS:GATK4_SELECTVARIANTS' {
         ext.prefix = { "${meta.id}_${intervals.simpleName}" }
-        ext.when   = { params.annotate_snv_switch && !(params.analysis_type == "wes")}
+        ext.when   = { !params.skip_snv_annotation && !(params.analysis_type == "wes")}
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply conf/modules/annotate_snvs.config:51]: You are right! It is redundant.. Its a hangover from when we didn't have controller at the subworkflow level. 😄

**Review by jemten — APPROVED** [review]
Looks good

**Pull request merged**